### ISSUE TITLE
False positive on SpacingAroundColon with context parameters

### ISSUE THREAD

**Issue by arcao** [detekt/detekt#8574]
## Expected Behavior
detekt should not warn about formatting

## Observed Behavior
detekt warns about missing space, but it's formatted correctly.

<img width="1090" height="257" alt="Image" src="https://github.com/user-attachments/assets/f96e5acb-0974-48f1-9138-e4a1a6e7321e" />

Auto-correct remove import, but not solve formatting issue.

## Steps to Reproduce
```kotlin
import com.example.Foo

context(foo: Foo)
fun doSomethingWithFoo() {
  foo.doSomething()
}
```

## Your Environment
* Version of detekt used: 1.23.8
* Version of Gradle used (if applicable): 8.13
* Version of IDE: Android Studio Narwhal Feature Drop | 2025.1.2
* Version of IDE detekt plugin: 2.4.3
* Operating System and version: MS Windows 11 Pro x64

**Comment by 3flex** [comment]
This would be fixed by the most recent updates to ktlint which are already included in `main` branch.

You might want to switch to snapshot builds until a new detekt release is out.
